Documentation/ApiOverview/MountPoints/Index.rst

```diff
@@ -1,174 +1,253 @@
+pairs of page IDs are added to the `MP` parameter forming a comma separated
+list. *Example:* `MP=13-23,84-26`.
 
-Example Pagetree:
 
-*  Root
+Slug generation
+---------------
 
-   *  Basic MountPoint
-   *  Company
+Since version v9 of TYPO3 `site handling ((link?)) <#>`__ and `slug generation
+((link?)) <#>`__ is built in. In TYPO3 a given page can only have *one* string
+for representation in a speaking URL. URLs nevertheless need to be unique and
+the `MP` get parameter should not be shown. TYPO3 will therefore combine the
+slug of the mount point page with the slugs of mounted pages according to some
+`((clever)) algorithm ((link?)) <#>`__.
 
-      *  About us
 
-MountPoint Configuration:
+Multisites
+==========
 
-:Title:
- Basic MountPoint
-:URL Segment:
- basic-mountpoint
-:Target Page:
- Company
-:Mount Point Display Option:
- "Show this page"
+((Make this paragraph simpler or more expressive or drop it.)) In cross domain
+sites the rendering context of each site is kept while only *the content* is
+retrieved from somewhere else. This ensures that the reader will never notice
+that content is reused and retrieved from a different site or a different
+branch of the page tree.
 
-Generated URLs:
 
-- `https://example.com/basic-mountpoint` -> shows content on this page
-- `https://example.com/basic-mountpoint/about-us` -> shows `company/about-us` content
-- `https://example.com/company/`
-- `https://example.com/company/about-us`
+Considerations and limitations
+==============================
 
-Basic MountPoint - "Show the mounted page"
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. ordered just alphabetically as there is no hierarchy
 
-Example Pagetree:
+Duplicated content and SEO
+   Mount points create "duplicate content" by definition. This may work against
+   search engine optimisation (SEO) and may need extra consideration.
 
+Menus and linking to mounted subpages
+   Think of it: Mounted pages don't have a representation of its own in the
+   page tree. This also means that you cannot have links to mounted subpages.
+   Instead menus can be used to make mounted subpages accessable. TYPO3 takes
+   care of the difficult task of proper menu generation.
 
-*  Root
+Multi language support
+   This is possible. All sites have to agree on the same meaning of language
+   IDs however.
 
-   *  Basic MountPoint
-   *  Company
+Slug uniqueness
+   Attention: *Slug uniqueness* is NOT garanteed for *multisites*.
 
-      *  About us
+   Here is an example to illustrate the reason: Let's say a mount point page
+   has the slug `/more` and mounts a page that has a subpage `imprint`. Let's
+   further assume the mount page has a sibling `more/imprint` which is a
+   regular page. Then there is a collision of a type that TYPO3 cannot detect.
+   The regular page will always work but the mounted subpage will never be
+   shown.
 
-MountPoint Configuration:
+   ((**TODO:** Though I've rewritten this passage I have to confess that I
```

Comment:
that's because the example is now wrong ;) (look at the title: it was about multisites, so the sibling case is part of the second tree)

Reply:
Maybe. But if I didn't easily understand - who else will?

Reply:
Added an additional example show-casing the potential conflict.